examples/pipelines/simple_pipeline/components/load_from_hub/src/main.py

```diff
@@ -46,9 +44,7 @@ def load(self, args):
         """
         # 1) Load data, read as Dask dataframe
         logger.info("Loading dataset from the hub...")
-        dataset = load_dataset(args.dataset_name, split="train")
-        dataset.to_parquet("data.parquet")
-        dask_df = dd.read_parquet("data.parquet")
+        dask_df = dd.read_parquet("hf://datasets/lambdalabs/pokemon-blip-captions/data/train-00000-of-00001-566cc9b19d7203f8.parquet")
```

Comment:
We still want to receive the dataset as an argument, right?

Reply:
Ok, fixed that